### DISCUSSION TITLE
Pybricks and Arduino

### DISCUSSION THREAD

**Discussion by didiersm** [pybricks/support#1390] [General]
Hi
I control a turnout with a servomotor connected to an Arduino and an Hc-06 bluetooth module.
it works very well from my smartphone with a bluetooth terminal.
how to define this equipment in pybricks.
I can't find how?
THANKS

**Comment by didiersm** [comment]
Hi 
No answers ?
No ideas?
Not possible?
Thks

**Comment by didiersm** [comment]
Hi
I'm trying to make progress on my Python-Arduino connection.
I achieved my goals with the PyBlueZ2 repository and it works quite well.
Well this repository does not exist with Pybricks and so my question is, how to transform this code so that it can work with Pybricks.
THANKS

Here my code working today:
```python
import bluetooth
from bluetooth import Protocols

bd_addr = "98:D3:xx:xx:xx:xx"

port = 1

server_sock = bluetooth.BluetoothSocket(Protocols.RFCOMM)

server_sock.connect((bd_addr, port))
#car = input("Quel est action ? ")
server_sock.send(b"B") #send caracter B  to the Arduino

server_sock.close()
```

**Reply by laurensvalk** [reply]
Which hub are you using?

**Reply by didiersm** [reply]
I want to use it with CityHub

**Reply by laurensvalk** [reply]
All of the Powered Up hubs with Pybricks use Bluetooth Low Energy (BLE) so no RFCOMM sockets (Bluetooth Classic).

Certain Arduinos have BLE, so you could technically implement the same hub-to-hub communication as on the hubs, but we don't have examples for that yet.

**Reply by didiersm** [reply]
Ok I have to find a BLE component.
I go ahead in this direction
Thks

**Reply by ziyidu** [reply]
Hello, excuse me, have you implemented this function now?
